Update Business docs (#11635)
readthedocs/readthedocs.org: docs/user/commercial/privacy-level.rst

Privacy Levels
--------------

.. include:: /shared/admonition-rtd-business.rst

Privacy levels allow you to set the visibility of your project and its versions.
This allows you to specify what information is available to the public and what is private.

> Privacy is generally controlled at the level of the version,

Reviewer: ```suggestion
Privacy can be controlled at the level of the project and each version,
```

I'm making this distinction here because the main content of the page talks about "Project privacy" and "Version privacy"

and nothing more granular (eg. specific directories or URLs) can be controlled.

Project privacy
~~~~~~~~~~~~~~~

By default, only users that belong to your organization can see the dashboard of your project and its builds.

> If you want users outside your organization to able to see the dashboard of your project,

Reviewer: ```suggestion
If you want users outside your organization to be able to see the dashboard of your project,
```

Not sure about this, but sound weird otherwise to me 🤷🏼

> and the build output of *public versions* you can set the privacy level of your project to ``Public``.

Reviewer: Maybe makes sense to explicitly say here this doesn't give access to the documentation. That will make this behavior clearer.

You can set the project privacy level in your :term:`dashboard` by navigating to :menuselection:`Admin --> Settings`
and changing :guilabel:`Privacy level` to `Public`.

Version privacy
~~~~~~~~~~~~~~~

Each version of your documentation can be set to either `Public` or `Private`.
This allows you to control who can see the documentation for a specific version.

> * Public versions are visible to everyone.

Reviewer: Same here. I think we should specify this is "Documentation for public versions" -- since the access to the dashboard and builds of this version will be defined by "Project privacy"

* Private versions are available only to people who have permissions to see them.
  They will not display on any list view, and will 404 when visited by people without viewing permissions.
  If you want to share your docs temporarily, see :doc:`/commercial/sharing`.

You can set the privacy level for each version in your :term:`dashboard` by navigating to :menuselection:`Versions --> $version ... Drop down --> Configure version` and changing :guilabel:`Privacy level` to `Public`.

Recommended workflow
~~~~~~~~~~~~~~~~~~~~

> For Business users,
> we recommend defaulting everything to private,

Reviewer: ```suggestion
We recommend defaulting everything to private,
```
